Makefile

```diff
@@ -35,7 +37,23 @@ kured-release-snapshot:
 	$(GORELEASER_CMD) release --clean --snapshot
 
 image: kured
-	$(SUDO) docker buildx build --load -t ghcr.io/$(DH_ORG)/kured:$(VERSION) .
+	$(SUDO) docker buildx build --no-cache --load -t ghcr.io/$(DH_ORG)/kured:$(VERSION) .
+
+dev-image: image
+	$(SUDO) docker tag ghcr.io/$(DH_ORG)/kured:$(VERSION) kured:latest
```

Comment:
would `canary` be a better tag than `latest` for an ephemeral dev image?

Reply:
I am fine with that.

Reply:
I went to "dev" instead of canary, as I think it makes it more explicit that it's not yet ready for release :)